app/javascript/packs/sdk.js

```diff
@@ -31,6 +31,13 @@ const runSDK = ({ baseUrl, websiteToken }) => {
       IFrameHelper.events.toggleBubble(state);
     },
 
+    toggleBubbleVisibility(visibility) {
+      document.querySelector('.woot--bubble-holder').style.display = visibility
```

Comment:
To be safe let us use here !!visibility and in the below line.

Reply:
Resolved in https://github.com/chatwoot/chatwoot/pull/4223/commits/dc0b790ed06d1f12e03e88d0bcee4f47096e57bb